### PR TITLE
[ADD] switch user via API

### PR DESCRIPTION
**Pull request by natacha-beck**

### CODE REVIEW

File: BrainPortal/app/controllers/users_controller.rb [hunk 1]
```diff
@@ -346,7 +372,10 @@ def switch #:nodoc:
     # ... but we must adjust the CBRAIN session object too
     cbrain_session.user_id = @user.id
 
-    redirect_to start_page_path
+    respond_to do |format|
```
Comment (prioux) [inline BrainPortal/app/controllers/users_controller.rb:375]: Why is this changed if 'switch' is not even an available API call ?

File: BrainPortal/app/controllers/users_controller.rb [hunk 2]
```diff
@@ -328,6 +329,31 @@ def destroy #:nodoc:
...
+  # Only use via API and accessible only to admin users
+  def create_user_session #:nodoc:
+    for_user_id = params[:id]
+    for_user    = User.find(for_user_id)
+
+    new_user_session = LargeSessionInfo.new(
+      user_id:    for_user_id,
+      session_id: SecureRandom.hex,
+      active: true,
+      data: {
```
Comment (prioux) [inline BrainPortal/app/controllers/users_controller.rb:341]: It's a bit weird that you are mixing the style

```ruby
:symbol => value
```

and

```ruby
symbol: value
```

**Review by prioux — CHANGES_REQUESTED** [review]
Minor changes.

**Comment by MontrealSergiy** [comment]
code seems ok, though not clear why the API is needed

**Comment by MontrealSergiy** [comment]
should I test it?

File: BrainPortal/app/controllers/users_controller.rb [hunk 3]
```diff
@@ -328,6 +329,31 @@ def destroy #:nodoc:
     end
   end
 
+  # Only use via API and accessible only to admin users
```
Comment (MontrealSergiy) [inline BrainPortal/app/controllers/users_controller.rb:332]: I would rephrase in polite  "Please, use only via API ..." 
or non-prescriptive manner "API-only action ..."
